extensions/wikia/TemplateDraft/controllers/TemplateClassificationController.class.php

```diff
@@ -33,13 +53,7 @@ private function getClassificationProp( $type ) {
 		return false;
 	}
 
-	public function classifyTemplate ($templateName, $type, bool $value, $actor = self::CLASSIFICATION_ACTOR_HUMAN ) {
-		$title = Title::newFromName( $titleName, NS_TEMPLATE );
-
-		if ( is_null( $title ) ) {
-			return false;
-		}
-
+	public function classifyTemplate ( $type, bool $value, $actor = self::CLASSIFICATION_ACTOR_HUMAN ) {
```

Comment:
This should still fatal due to the bool type, is this working on a devbox?

Also, minor coding convention fix while you're in there, there shouldn't be a space between the method name and the left parenthesis.